ada_2_fhir-r4/zibs2020/fhir_instance/nl-core-DevelopmentChild-01.xml

```diff
@@ -0,0 +1,135 @@
+   <code>
+      <coding>
+         <system value="http://snomed.info/sct"/>
+         <code value="251803000"/>
+         <display value="bevinding betreffende ontwikkeling van kind"/>
+      </coding>
+   </code>
+   <subject>
+      <reference value="Patient/nl-core-Patient-01"/>
+      <type value="Patient"/>
+      <display value="Patient, Johanna Petronella Maria (Jo) van Putten-van der Giessen"/>
+   </subject>
+   <effectiveDateTime value="2016-12-19"/>
+   <note>
+      <text value="Kind is heel beweeglijk"/>
+   </note>
+   <component>
+      <code>
+         <coding>
+            <system value="http://snomed.info/sct"/>
+            <code value="275819003"/>
+            <display value="blaascontrole bij kind"/>
+         </coding>
+      </code>
+      <valueCodeableConcept>
+         <coding>
+            <system value="http://snomed.info/sct"/>
+            <code value="26411000146101"/>
+            <display value="geen blaascontrole op kinderleeftijd"/>
+         </coding>
+      </valueCodeableConcept>
+   </component>
+   <component>
+      <code>
+         <coding>
+            <system value="http://snomed.info/sct"/>
+            <code value="275818006"/>
+            <display value="controle over defecatie op kinderleeftijd"/>
+         </coding>
+      </code>
+      <valueCodeableConcept>
+         <coding>
+            <system value="http://snomed.info/sct"/>
+            <code value="26421000146106"/>
+            <display value="geen controle over defecatie op kinderleeftijd"/>
+         </coding>
+      </valueCodeableConcept>
+   </component>
+   <component>
+      <code>
+         <coding>
+            <system value="http://loinc.org"/>
+            <code value="42798-9"/>
+            <display value="Age at menarche"/>
+         </coding>
+      </code>
+      <valueQuantity>
+         <value value="16"/>
+         <unit value="a"/>
+         <system value="http://unitsofmeasure.org"/>
+         <code value="a"/>
+      </valueQuantity>
+   </component>
+   <component>
+      <code>
+         <coding>
+            <system value="http://snomed.info/sct"/>
+            <code value="282716004"/>
+            <display value="grove motoriek"/>
+         </coding>
+      </code>
+      <valueCodeableConcept>
+         <coding>
+            <system value="http://snomed.info/sct"/>
+            <code value="282888002"/>
+            <display value="in staat om zich op te trekken tot staan vanuit zithouding"/>
+         </coding>
+      </valueCodeableConcept>
+   </component>
+   <component>
+      <code>
+         <coding>
+            <system value="http://snomed.info/sct"/>
+            <code value="225597007"/>
+            <display value="vermogen tot interactie met anderen"/>
+         </coding>
+      </code>
+      <valueCodeableConcept>
+         <coding>
+            <system value="http://snomed.info/sct"/>
+            <code value="26281000146106"/>
+            <display value="in staat om te geven en nemen"/>
```

Comment:
Correct display value from SNOMED browser: "in staat om 'geven en nemen' te spelen" (with the quotes)

Reply:
This value is already present in the ADA instance, and I'd rather not change those manually, relative to how they are generated from ART-DECOR.